things/service.go

```diff
@@ -439,3 +477,38 @@ func (ts *thingsService) members(ctx context.Context, token, groupID, groupType
 	}
 	return res.Members, nil
 }
+
+func (ts *thingsService) authorize(ctx context.Context, subject, object string, relations ...string) error {
+	req := &mainflux.AuthorizeReq{
+		Sub: subject,
+		Obj: object,
+	}
+	for _, relation := range relations {
```

Comment:
since, for now, we will check only one relation and we should strive that `keto` does evaluation for more complex relations by carefuly designing policies you should remove this loop
also this can slow server to have relations in evaluated in loop